Fundamentals/Backoffice/LogViewer/index.md

```diff
@@ -35,124 +35,94 @@ With the flexibility of Umbraco, we give you the power to implement your own `IL
-            var tableClient = cloudStorage.CreateCloudTableClient();
-            CloudTable table = tableClient.GetTableReference("LogEventEntity");
-
-            var logs = new List<LogEvent>();
-            var count = 0;
-
-            // Create the table query
-            // TODO: Use a range query to filter by start & end date on the Timestamp
-            TableContinuationToken token = null;
-            var results = new List<LogEventEntity>();
-            do
-            {
-                var queryResult = table.ExecuteQuerySegmentedAsync(new TableQuery<LogEventEntity>(), token).GetAwaiter().GetResult();
-                results.AddRange(queryResult.Results);
-                token = queryResult.ContinuationToken;
-            } while (token != null);
-
-            // Loop through the results, displaying information about the entity.
-            foreach (var entity in results.Skip(skip))
-            {
-                // Reads the compact JSON format stored in the 'Data' column back to a LogEvent
-                // Same as the JSON txt files does
-                var logItem = LogEventReader.ReadFromString(entity.Data);
-
-                if (count > skip + take)
-                {
-                    break;
-                }
-
-                if (count < skip)
-                {
-                    count++;
-                    continue;
-                }
-
-                if (filter.TakeLogEvent(logItem))
-                {
-                    logs.Add(logItem);
-                }
-
-                count++;
-            }
-
-            return logs;
-        }
-
-        public override IReadOnlyList<SavedLogSearch> GetSavedSearches()
-        {
-            //TODO: Fetch from Azure Table
-            return base.GetSavedSearches();
-        }
-
-        public override IReadOnlyList<SavedLogSearch> AddSavedSearch(string name, string query)
-        {
-            //TODO: Add to Azure Table
-            return base.AddSavedSearch(name, query);
-        }
-
-        public override IReadOnlyList<SavedLogSearch> DeleteSavedSearch(string name, string query)
-        {
-            //TODO: Remove from Azure Table
-            return base.DeleteSavedSearch(name, query);
-        }
+        //TODO: Add to Azure Table
+        return base.AddSavedSearch(name, query);
     }
+
+    public override IReadOnlyList<SavedLogSearch>? DeleteSavedSearch(string? name, string? query)
+    {
+        //TODO: Remove from Azure Table
+        return base.DeleteSavedSearch(name, query);
+    }
+}
+
+public class AzureTableLogEntity : LogEventEntity, ITableEntity
+{
+    public DateTimeOffset? Timestamp { get; set; }
+
+    public ETag ETag { get; set; }
+}
 ```
 
+Do note that we have to implement our own version of a `LogEventEntity` this is beacuse the `TableClient` needs whatever it's fetching to implement the `ITableEntity` interface.  
```

Comment:
```suggestion
Keep in mind that we have to implement our own version of a `LogEventEntity`. This is because the `TableClient` needs whatever it is fetching to implement the `ITableEntity` interface.  
```